src/tile/tile.scss

```diff
@@ -211,10 +211,24 @@
       position: absolute;
       bottom: $iui-baseline * 0.5;
       right: $iui-s;
-      visibility: hidden;
+
+      clip-path: inset(50%);
+      overflow: hidden;
+      white-space: nowrap;
+      height: 1px;
+      width: 1px;
+      padding: 0;
+      margin: -1px;
+      border-width: 0;
 
       &.iui-visible {
```

Comment:
When is this `iui-visible` class applied? The more-options button should really only be shown when the tile is hovered or has focus within.

Reply:
I think we added this class to show more-options when it is clicked and dropdown is shown. 
![image](https://user-images.githubusercontent.com/36186912/152940730-cfe5b2c3-8e7d-4407-8e96-41a32584932c.png)
Without this class:
![image](https://user-images.githubusercontent.com/36186912/152940812-24b02c0f-51e5-4023-aa78-2f97a0b8b39b.png)

Reply:
Oh, for some reason I thought this was a new addition. In that case, we should undo the visually hidden styles inside this class

Reply:
`.iui-visible` is back.